core/src/main/kotlin/dev/hotwire/core/turbo/webview/HotwireWebView.kt

```diff
@@ -31,7 +31,7 @@ open class HotwireWebView @JvmOverloads constructor(
 ) : WebView(context, attrs) {
     private val gson = GsonBuilder().disableHtmlEscaping().create()
 
-    var elementTouchIsScrollable = false
+    var elementTouchPreventsPullsToRefresh = false
```

Comment:
I've renamed `elementTouchIsScrollable` to `elementTouchPreventsPullsToRefresh `, to align with the new functionality. If you feel like this is not a good name, or not necessary, I'm happy to change it back.